thriftbp/thrifttest/server_example_test.go

```diff
@@ -0,0 +1,60 @@
+package thrifttest_test
+
+import (
+	"context"
+	"testing"
+
+	baseplatethrift "github.com/reddit/baseplate.go/internal/gen-go/reddit/baseplate"
+	"github.com/reddit/baseplate.go/secrets"
+	"github.com/reddit/baseplate.go/thriftbp/thrifttest"
+)
+
+type mockedBaseplateService struct {
+	Fail bool
+	Err  error
+}
+
+func (srv mockedBaseplateService) IsHealthy(ctx context.Context, req *baseplatethrift.IsHealthyRequest) (r bool, err error) {
+	return !srv.Fail, srv.Err
+}
+
+// In real test this function needs to be named TestService instead,
+// but doing that will break this example.
+func ServiceTest(t *testing.T) {
+	// Initialize this properly in a real test
+	var store *secrets.Store
```

Comment:
Should we reference here that this should be provided through `baseplate.New`?

Reply:
added mention of `secrets.NewTestSecrets` here.